main.py

```diff
@@ -38,7 +39,22 @@ def main():
         config_file = networkconfig[conf]
         with tf.Session() as sess:
             builder = ModelBuilder(config_file, sess)
+            builder.add_input_layer()
+
+            # Add a number of hidden layers
+            for _ in range(config_file[HIDDEN_LAYERS]):
+                builder.add_layer(config_file[HIDDEN_NEURONS])
+
+            if builder._model.use_pretrained_net:
+                builder.add_secondary_output()
+                builder.add_layer(config_file[HIDDEN_NEURONS])
+
+            builder.add_output_layer()\
+                .add_precision_operations()
+
             network_model = builder.build()
+            if config_file[USE_PRETRAINED_NET]:
```

Comment:
Can't this be moved to the model itself? To the beginning of the train function maybe? Just tryting to keep the main.py file as clean as possible :smile: